artiq/coredevice/ad9910.py

```diff
@@ -465,7 +465,7 @@ def set_cfr2(self,
                      (sync_validation_disable << 5))
 
     @kernel
-    def init(self, blind: TBool = False):
+    def init(self, blind: TBool = False, dds_channel_idx: TInt32 = 0):
```

Comment:
Can we get rid of the ``blind`` parameter for both ad9910 and Urukul CPLD?

(nvm: it still needs to be there for proto rev 8)